### PR TITLE
changed the all capital string value in string.xml

### PR DESCRIPTION
**Pull request by sherlockbeard**
changed the all capital string value to normal capitalized values . and applied textallcap to the required places .
#2935  and #2873

### CODE REVIEW

File: app/src/main/res/values/strings.xml [hunk 1]
```diff
@@ -279,17 +279,17 @@
...
   <string name="notifications_thank_you_edit">Thank you for making an edit</string>
   <string name="notifications_mention">%1$s mentioned you on %2$s.</string>
   <string name="toggle_view_button">Toggle view</string>
-  <string name="nearby_directions">DIRECTIONS</string>
-  <string name="nearby_wikidata">WIKIDATA</string>
-  <string name="nearby_wikipedia">WIKIPEDIA</string>
+  <string name="nearby_directions">Directions</string>
+  <string name="nearby_wikidata">Wikidata</string>
+  <string name="nearby_wikipedia">Wikipedia</string>
   <string name="nearby_commons">COMMONS</string>
```
Comment (neslihanturan) [inline app/src/main/res/values/strings.xml]: Hi, it seems like this one is forgotten

Comment (sherlockbeard) [reply app/src/main/res/values/strings.xml]: @neslihanturan  the change you requested is done

File: app/src/main/res/values/strings.xml [hunk 2]
```diff
@@ -279,18 +279,18 @@
   <string name="nearby_location_has_not_changed">Location has not changed.</string>
   <string name="nearby_location_not_available">Location not available.</string>
   <string name="location_permission_rationale_nearby">Permission required to display a list of nearby places</string>
-  <string name="get_directions">GET DIRECTIONS</string>
+  <string name="get_directions">Get Directions</string>
   <string name="read_article">READ ARTICLE</string>
```
Comment (neslihanturan) [inline app/src/main/res/values/strings.xml]: This one also needs to be done, I am not able to check whole file now. Can you please cross over it one more time?

File: app/src/main/res/values/strings.xml [hunk 3]
```diff
@@ -65,7 +65,7 @@
   <string name="refresh_button">Refresh</string>
   <string name="display_list_button">List</string>
   <string name="gps_disabled">GPS is disabled in your device. Would you like to enable it?</string>
-  <string name="enable_gps">Enable GPS</string>
+  <string name="enable_gps">Enable Gps</string>
```
Comment (neslihanturan) [inline app/src/main/res/values/strings.xml]: GPS is stands for Global Positioning System. Please do not change abbreviations like GPS, FAB etc. Just make sure there are no normal word which is written in all caps.

Comment (neslihanturan) [reply app/src/main/res/values/strings.xml]: And I didn't understand why you changed just one GPS while other GPS is still all caps, I would expect all GPSs are written in the same way. Consistency is important. In our case please leave all GPSs as all capital though.

File: app/src/main/res/values/strings.xml (hunk 2 above)
Comment (sherlockbeard) [reply app/src/main/res/values/strings.xml]: i cross checked and all of them are done now

File: app/src/main/res/values/strings.xml [hunk 4]
```diff
@@ -279,18 +279,18 @@
   <string name="nearby_location_has_not_changed">Location has not changed.</string>
   <string name="nearby_location_not_available">Location not available.</string>
   <string name="location_permission_rationale_nearby">Permission required to display a list of nearby places</string>
-  <string name="get_directions">GET DIRECTIONS</string>
-  <string name="read_article">READ ARTICLE</string>
+  <string name="get_directions">Get Directions</string>
+  <string name="read_article">Read Article</string>
```
Comment (neslihanturan) [inline app/src/main/res/values/strings.xml]: At where do you capitalize them in xml layouts? I think you have forgotten capitalize these two with android:textAllCaps="true"

File: app/src/main/res/values/strings.xml [hunk 5]
```diff
@@ -327,10 +327,10 @@
   <string name="provider_searches">Recently searched queries</string>
   <string name="error_loading_categories">Error occurred while loading categories.</string>
   <string name="error_loading_subcategories">Error occurred while loading subcategories.</string>
-  <string name="search_tab_title_media">MEDIA</string>
-  <string name="search_tab_title_categories">CATEGORIES</string>
-  <string name="explore_tab_title_featured">FEATURED</string>
-  <string name="explore_tab_title_mobile">UPLOADED VIA MOBILE</string>
+  <string name="search_tab_title_media">Media</string>
```
Comment (neslihanturan) [inline app/src/main/res/values/strings.xml]: Did you capitalize them programatically?

File: app/src/main/res/values/strings.xml [hunk 6]
```diff
@@ -327,10 +327,10 @@
...
+  <string name="search_tab_title_categories">Categories</string>
```
Comment (neslihanturan) [inline app/src/main/res/values/strings.xml]: Did you capitalize them programatically?

File: app/src/main/res/values/strings.xml [hunk 7]
```diff
@@ -327,10 +327,10 @@
...
+  <string name="explore_tab_title_featured">Featured</string>
```
Comment (neslihanturan) [inline app/src/main/res/values/strings.xml]: Did you capitalize them programatically?

File: app/src/main/res/values/strings.xml [hunk 8]
```diff
@@ -327,10 +327,10 @@
...
+  <string name="explore_tab_title_mobile">Uploaded via mobile</string>
```
Comment (neslihanturan) [inline app/src/main/res/values/strings.xml]: Did you capitalize them programatically?

File: app/src/main/res/values/strings.xml (hunk 3 above)
Comment (sherlockbeard) [reply app/src/main/res/values/strings.xml]: ok i change is done

File: app/src/main/res/values/strings.xml [hunk 9]
```diff
@@ -366,14 +366,14 @@
   <string name="delete_search_dialog">Do you want to delete this search?</string>
   <string name="search_history_deleted">Search history deleted</string>
   <string name="nominate_delete">Nominate For Deletion</string>
-  <string name="delete">DELETE</string>
+  <string name="delete">Delete</string>
```
Comment (neslihanturan) [inline app/src/main/res/values/strings.xml]: Did you capitalize them programatically?

File: app/src/main/res/values/strings.xml [hunk 10]
```diff
@@ -366,14 +366,14 @@
...
 
   <string name="Achievements">Achievements</string>
-  <string name="statistics">STATISTICS</string>
+  <string name="statistics">Statistics</string>
```
Comment (neslihanturan) [inline app/src/main/res/values/strings.xml]: Did you capitalize them programatically?

File: app/src/main/res/values/strings.xml [hunk 11]
```diff
@@ -366,14 +366,14 @@
...
   <string name="statistics_thanks">Thanks Received</string>
   <string name="statistics_featured">Featured Images</string>
   <string name="statistics_wikidata_edits">Images via \"Nearby Places\"</string>
-  <string name="level">LEVEL</string>
+  <string name="level">Level</string>
```
Comment (neslihanturan) [inline app/src/main/res/values/strings.xml]: Did you capitalize them programatically?

File: app/src/main/res/values/strings.xml [hunk 12]
```diff
@@ -537,7 +537,7 @@ Upload your first media by tapping on the add button.</string>
   <string name="previous_button_tooltip_message">Click to reuse the title and description you entered in your previous picture and modify it to fit your current one</string>
   <string name="welcome_do_upload_content_description">Examples of good images to upload to Commons</string>
   <string name="welcome_dont_upload_content_description">Examples of images not to upload</string>
-  <string name="skip_image">SKIP THIS IMAGE</string>
+  <string name="skip_image">Skip this image</string>
```
Comment (neslihanturan) [inline app/src/main/res/values/strings.xml]: Did you capitalize them programatically?

File: app/src/main/res/values/strings.xml (hunk 4 above)
Comment (sherlockbeard) [reply app/src/main/res/values/strings.xml]: thing is they are not used anywhere.

File: app/src/main/res/values/strings.xml (hunk 12 above)
Comment (sherlockbeard) [reply app/src/main/res/values/strings.xml]: no but i  checked all through regular expression

Comment (neslihanturan) [reply app/src/main/res/values/strings.xml]: What do you mean by "checked all through regular expression" can you please explain?

**Comment by neslihanturan** [comment]
@sherlockbeard localisation updates commit shouldn't be in your pull request. Additionally please make sure you checked diff of your changes. There shouldn't be any extra code changes other than required ones. Adding new imports and spaces or line breaks included. Your changes should look clean so that we can review it easily. Please ping me when your pull request is ready after all your changes are clean and my reviews are considered, so that I can merge. Thanks for your cooperation:)

**Comment by sherlockbeard** [comment]
can i start a new pull request .

**Comment by sherlockbeard** [comment]
i will close this branch because mistakenly i forced pushed other thing . i am sorry for it

**Comment by neslihanturan** [comment]
Sure, a clean pull request is always okay  @sherlockbeard , please mention me when your  PR is ready :)

File: app/src/main/res/values/strings.xml (hunk 12 above)
Comment (sherlockbeard) [reply app/src/main/res/values/strings.xml]: sorry for the late reply i checked them through the method shown in the screenshot .

**Pull request closed without merging**